main/admin.py

```diff
@@ -3,13 +3,31 @@
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin
 
-from .models import ActivityCode, Department, Project, User
+from .models import ActivityCode, Department, Funding, Project, User
 
 admin.site.register(User, UserAdmin)
 admin.site.register(Department)
 admin.site.register(ActivityCode)
 
 
+@admin.register(Funding)
+class FundingAdmin(admin.ModelAdmin):  # type: ignore [type-arg]
+    """Admin class for the Funding model."""
+
+    list_display = (
+        "project",
+        "project_code",
+        "funding_body",
+        "source",
+        "activity_code",
+        "expiry_date",
+        "budget",
+        "daily_rate",
+        "effort",
+        "effort_left",
+    )
```

Comment:
We do not need to include all of them here. I'd suggest removing a couple of them:
```suggestion
    list_display = (
        "project",
        "project_code",
        "funding_body",
        "source",
        "expiry_date",
        "budget",
        "effort",
        "effort_left",
    )
```